doc/Motr_LNet_Transport.rst

```diff
@@ -0,0 +1,625 @@
+==============================
+HLD OF Motr LNet Transport
+==============================
+
+This document presents a high level design (HLD) of the Motr LNet Transport. The main purposes of this document are: (i) to be inspected by M0 architects and peer designers to ascertain that high level design is aligned with M0 architecture and other designs, and contains no defects, (ii) to be a source of material for Active Reviews of Intermediate Design (ARID) and detailed level design (DLD) of the same component, (iii) to serve as a design reference document.
```

Comment:
I see you have replaced the old name with new "Motr".
Please also replace "M0" with "Motr" in this paragraph.

Reply:
Incorporated